### ISSUE TITLE
Migration from feather-vue to lucide-vue

### ISSUE THREAD

**Issue by queenvictoria** [lucide-icons/lucide#400]
I am cutting over from feather to lucide for our vue project.

In [vue-feather-icons](https://github.com/egoist/vue-feather-icons) I was able to do this (in pug):

```pug
    trash-icon(@click="foo()")
```

So in lucide-vue/nuxt I have this:
```pug
    icon-trash(@click="foo()")
```

I notice that the click handler is not called. Is there something I'm missing? @venashial ?

**Comment by ericfennis** [comment]
Nice catch, never checked this, I will test maybe the onclick handler will not pass to the functional component

**Comment by ericfennis** [comment]
Fixed it in v0.16.1 in `lucide-vue`.
Thanks for reporting this issue!
